src/docs/ios.md

```diff
@@ -17,81 +17,65 @@ and phystical `devices`.
+![](../images/screenshots/prov-prof-1.png)
+1. Type you email in the User Email Adress field. Other form elements are optional.
+![](../images/screenshots/prov-prof-2.png)
+1. Choose Request is Saved to Disc and save your certificate request.
+1. [Create][apple-dev-certificates-add] a new iOS Distribution (App Store and Ad Hoc) certificate on the Apple Developers website.
+1. Choose a certificate request you've created 2 steps earlier.
+1. Download your certificate and double click to add it to your Keychain.
 
 #### Provisioning Profiles
 
-When you run `op . -ios` on your project for the first time, you may see the
+When you run `ssc compile --target=ios .` on your project for the first time, you may see the
 following because you don't have a [provisioning profile][A5]...
 
 ```
-op . -ios
-- security: unable to open "./distribution.mobileprovision" for reading: No such file or directory
-- failed to extract uuid from provisioning profile +30ms
+ssc compile --target=ios .
+• provisioning profile not found: /Users/chicoxyzzy/dev/socketsupply/birp/./distribution.mobileprovision. Please specify a valid provisioning profile in the ios_provisioning_profile field in your ssc.config
 ```
 
-You will need your device id. You can get it by plugging in your iPhone and then
-running the following command.
-
-```
-op -mid
-00004201-101371260222221F
-```
-
-1. Create a provisioning profile for `Development` (build and deploy to your
-iphone). Include the certificate for `iOS Development`.
-
-2. Create a profile for `Ad-Hoc` distribution (give other people copies that
-they can install). Include the devices you want to deploy to and include the
-certificate you created for `iOS Distribution`.
-
-3. Download the profiles and add them to your project. They are
-secret, don't commit them, add the appropriate file names to your `.gitignore`
-file. op will pick the right one for you based on your build flags.
+1. [Create][apple-dev-profiles-add] a new Ad Hoc profile. Use the App ID you created with the wildcard.
+1. Pick the certificate that you added to your Keychain two steps earlier.
+1. Add the devices that the profile will use.
+1. Add a name for your new distribution profile (we recommend to name it "distribution").
+1. Download the profile and double click it. This action will open Xcode. You can close it after it's completely loaded.
+1. Place your profile to your project directory (same directory as ssc.config). *The profiles are secret, add your profile to `.gitignore`*.
 
 ### Configuration
 
-Edit the following fields in the `ssc.config` file of your project.
-
-```settings
-apple_teamId: Z3M838H537
-apple_debug_distribution_method: ad-hoc
-apple_debug_signing_certificate: iOS Distribution (App Store and Ad Hoc)
-apple_debug_provisioning_profile: src/development.mobileprovision
+1. Set the "ios_team_id" value in `ssc.config` to the value of Team ID [here][https://developer.apple.com/account/#!/membership/]
```

Comment:
this one is hacky, since I rely on the fact that now it redirects to the proper team